src/guid.cpp

```diff
@@ -21,14 +21,31 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <sstream>
 
 #include "serverenvironment.h"
+#include "util/base64.h"
+
+GUIdGenerator::GUIdGenerator()
+{
+	m_rand_usable = m_rand.entropy() > 0.01;
+}
 
 GUId GUIdGenerator::next(const std::string &prefix)
 {
 	std::stringstream s_guid;
 
-	s_guid << prefix << std::hex << m_next << '-' << std::hex << m_env->getGameTime();
-
-	m_next++;
+	std::mt19937_64 gen(m_rand());
+	std::uniform_int_distribution<unsigned long long> m_uniform(0, UINT64_MAX);
```

Comment:
Why `unsigned long long` and not `u64`?

Reply:
Because of trying process. I will fix it if this solution is accepted. Or write it differently if not.